testing_and_setup/compass/ocean/global_ocean/scripts/cull_mesh.py

```diff
@@ -37,16 +37,37 @@
 
 
 parser = OptionParser()
-parser.add_option("--with_cavities", action="store_true", dest="with_cavities")
+parser.add_option("--with_cavities", action="store_true", dest="with_cavities",
+                  help="Whether the mesh should include Antarctic ice-shelf"
+                       " cavities")
 parser.add_option("--with_critical_passages", action="store_true",
-                  dest="with_critical_passages")
+                  dest="with_critical_passages",
+                  help="Whether the mesh should open the standard critical "
+                       "passages and close land blockages from "
+                       "geometric_features")
+parser.add_option("--custom_critical_passages", dest="custom_critical_passages",
+                  help="A geojson file with critical passages to open.  This "
+                       "file may be supplied in addition to or instead of "
+                       "the default passages (--with_critical_passages)")
+parser.add_option("--custom_land_blockages", dest="custom_land_blockages",
+                  help="A geojson file with critical land blockages to close. "
+                       "This file may be supplied in addition to or instead of "
+                       "the default blockages (--with_critical_passages)")
 parser.add_option("--preserve_floodplain", action="store_true",
-                  dest="preserve_floodplain", default=False)
+                  dest="preserve_floodplain", default=False,
+                  help="Whether to to use the cellSeedMask field in the base "
+                       "mesh to preserve a floodplain at elevations above z=0")
```

Comment:
@pwolfram, can you tell me if this is correct?  I think it is based on my recent work on `inject_preserve_floodpalin`.

Reply:
@pwolfram any comment? It is just a help text, so I may merge to get this in.